src/api-stuff.js

```diff
@@ -0,0 +1,20 @@
+const myGameId = 'm4sf6ZHY83BiDA7soyq9';
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const scoresURL = `${baseURL}games/${myGameId}/scores/`;
+
+async function postScore(data) {
+  const res = await fetch(scoresURL, {
+    method: 'POST',
+    mode: 'cors',
+    headers: { 'Content-type': 'application/json' },
+    body: JSON.stringify(data),
+  });
+  return res.json(); // parses JSON response into native JavaScript objects
```

Comment:
- [ ] Please use the Arrow function as the requirements say.

![olo](https://user-images.githubusercontent.com/61562626/137162062-9946651a-69b2-4c4d-84f8-d6ce740d2a66.jpeg)